package.json

```diff
@@ -24,7 +24,7 @@
     "build:ag-grid-theme": "cd packages/ag-grid-theme && yarn build",
     "test": "modular test",
     "test:components": "cypress run --component --browser chrome --headless",
-    "test:components:local": "cypress open --component --browser chrome",
+    "test:components:local": "cypress open --component --browser electron",
```

Comment:
Could we create a new PR for this please

Reply:
Ah, yes, forgot about this